libs/wingsdk/src/shared-aws/bucket.inflight.ts

```diff
@@ -27,13 +29,13 @@ export class BucketClient implements IBucketClient {
    * @param key Key of the object
    */
   public async exists(key: string): Promise<boolean> {
-    const command = new ListObjectsV2Command({
+    const command = new HeadObjectCommand({
```

Comment:
Do the AWS permissions need to be updated at all? see here https://github.com/winglang/wing/blob/f8a5abf8a7a6a143f3e089d1fbe2c2ef47904bfc/libs/wingsdk/src/shared-aws/permissions.ts#L84

Reply:
Yes, in order to retrieve the object's metadata, the `HeadObjectCommand` requires also the `s3:GetObject` and `s3:GetObjectAttributes` permissions. Thanks for reminding me, I've fixed it now.